services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableSwitchRulesSpec.groovy

```diff
@@ -0,0 +1,73 @@
+package org.openkilda.functionaltests.spec.multitable
+
+import static org.openkilda.testing.Constants.RULES_INSTALLATION_TIME
+
+import org.openkilda.functionaltests.HealthCheckSpecification
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.model.Cookie
+
+class MultitableSwitchRulesSpec extends HealthCheckSpecification {
```

Comment:
Please add a narrative that explains that we can switch between switch modes and the rules will be automatically re-installed if there are now flows on the switch

Reply:
The 'See' annotation was added